README.md

```diff
@@ -112,10 +112,18 @@ of random tech startup business names.
 2. **Set OpenAI API Key**: Replace `$YOUR_OPENAI_API_KEY` with your
    actual OpenAI API key.
 
+    On macOS or Linux systems,
+
     ```bash
     export OPENAI_API_KEY=$YOUR_OPENAI_API_KEY
     ```
 
+    On Windows systems,
+
```

Comment:
Thank you for adding the powershell command! Could you add the terminal command as well for windows user?

Reply:
I believe is 'set OPENAI_API_KEY=your_openai_api_key'

Reply:
setx is persistent and should work for both cmd and powershell. This should be fine. https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/setx